src/mlabns/mlabns_impl.hpp

```diff
@@ -0,0 +1,114 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
```

Comment:
I don't understand why this file is not reported as being a copy with small modifications of the original `.cpp` file. Since the changes between the two should be very small.